test/e2e/tests/test_models_endpoint.py

```
@@ -292,60 +293,20 @@ def test_single_subscription_auto_select(self):
             # Wait for subscription to reconcile before creating API key
             _wait_for_maas_subscription_phase(subscription_name, namespace=maas_ns)
 
+            # Wait for model to become Ready after governance pairing is created
+            log.info(f"Waiting for model to reconcile and become Ready...")
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)